extdata/example_data.csv

```diff
@@ -0,0 +1,85 @@
+date,earnings
+1960 Q1,0.71
+1960 Q2,0.63
+1960 Q3,0.85
+1960 Q4,0.44
+1961 Q1,0.61
+1961 Q2,0.69
+1961 Q3,0.92
+1961 Q4,0.55
+1962 Q1,0.72
+1962 Q2,0.77
+1962 Q3,0.92
+1962 Q4,0.6
+1963 Q1,0.83

```

Comment:
No need to add this `.csv`. An outside user won't have access to this anyway. define a toy dataset based on this one instead. Use just the first few rows.